syncd/syncd_flex_counter.cpp

```diff
@@ -836,16 +836,29 @@ bool FlexCounter::isEmpty()
 {
     SWSS_LOG_ENTER();
 
+    return isIdsEmpty() && isPluginsEmpty();
+}
+
+bool FlexCounter::isIdsEmpty()
```

Comment:
areIdsEmpty ? since multipe ids maps we check here ?

Reply:
Agree with @kcudnik. Same for the plugins. Since the functions return true if **all** IDs/plugins are empty, I suggest `allIdsEmpty()` and `allPluginsEmpty()`